ApiDocs.Validation/Params/TestSetupRequestDefinition.cs

```diff
@@ -121,6 +121,13 @@ public override ValidationError[] CheckForErrors()
 
             MethodDefinition.AddTestHeaderToRequest(scenario, request);
 
+            // parse the passed in addtional headers and add to request..format for headers should be <HeaderName>:<HeaderValue>
+            foreach (string nameValueHeader in account.AdditionalHeaders)
```

Comment:
Instead of copying this code in two places, refactor this into a method similar to AddTestHeaderToRequest() and call it from these two places.